File: CS/CustomReminderExample/NumberToTimeSpanConverter.cs
using System;
using System.Globalization;
using System.Windows.Data;

namespace CustomReminderExample {
    public class TimeSpanToNumberConverter : IValueConverter {
        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) {
            return ((TimeSpan)value).Minutes;
        }

        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) {
            return TimeSpan.FromMinutes(System.Convert.ToDouble(value));
        }
    }
}

File: VB/CustomReminderExample/NumberToTimeSpanConverter.vb
Imports System
Imports System.Globalization
Imports System.Windows.Data

Namespace CustomReminderExample

    Public Class TimeSpanToNumberConverter
        Implements IValueConverter

        Public Function Convert(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As CultureInfo) As Object Implements IValueConverter.Convert
            Return CType(value, TimeSpan).Minutes
        End Function

        Public Function ConvertBack(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As CultureInfo) As Object Implements IValueConverter.ConvertBack
            Return TimeSpan.FromMinutes(System.Convert.ToDouble(value))
        End Function
    End Class
End Namespace
